src/rascal_utility.cc

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file   rascal_utility.cc
+ *
+ * Copyright  2019 COSMO (EPFL), LAMMM (EPFL)
+ *
+ * Rascal is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3, or (at
+ * your option) any later version.
+ *
+ * Rascal is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; see the file LICENSE. If not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+#include <cstdlib>
+#include <fstream>
+#include <iterator>
+
+#include "rascal_utility.hh"
+
+#ifdef __GNUG__
+#include <cxxabi.h>
+#endif
+
+std::string rascal::internal::type_name_demangled(const char * name) {
+#ifdef __GNUG__
+  int status = 0;
```

Comment:
```suggestion
  int status{0};
```

Reply:
Could this be enforced with a cpplint/clang-format rule instead of having to manually check and change it?

Reply:
I'll have a look.

Reply:
Does not seem to have the option, neither clang-format nor cpplint.
I would file this under: Whatever gets caught, will should change but we try not to be to picky about it. If it's caught in review, so be it. If not, well we have no ways to check automatically anyways.